src/command_modules/azure-cli-acr/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.0.22
+++++++
+* Improve az acr repository command group
```

Comment:
Might be good to add some specific wording around deleting/untagging so we know what those improvements are by a glance.

Reply:
Yes, these entries should be as specific as is reasonable.
`* `acr repository`: Added support for `--image` to create/show/.etc commands`